dotnet/src/Extensions/Planning.ActionPlanner/ActionPlanner.cs

```diff
@@ -126,7 +126,7 @@ public async Task<Plan> CreatePlanAsync(string goal)
         {
             if (p.Value != null)
             {
-                plan.State[p.Key] = p.Value.ToString();
+                plan.Parameters[p.Key] = p.Value.ToString();
```

Comment:
We'll need to change this in CP Chat WebApp whenever the nuget's are updated since it's checking for state. Can you add a TODO in PlanViewer?